go/pkg/amqp/url_test.go

```diff
@@ -32,13 +35,6 @@ func ExampleParseURL() {
 		"amqps://host",
 		"/path",
 		"",
-		":1234",
-                // Taken out because the go 1.4 URL parser isn't the same as later
```

Comment:
Go 1.4 or the latest Go 1.14 in this case?

Reply:
It is an old comment. It is really about go 1.4 when the test apparently broke for the first time, due to parsing differences.